attestation/yubiattest/signature_test.go

```diff
@@ -52,11 +52,7 @@ func TestCreateSelfSignedCertificate(t *testing.T) {
 	for _, test := range tests {
 		commonName := "test.example.com"
 		template := x509.Certificate{
-			// SerialNumber is negative to ensure that negative
-			// values are parsed. This is due to the prevalence of
-			// buggy code that produces certificates with negative
-			// serial numbers.
-			SerialNumber: big.NewInt(-1),
+			SerialNumber: big.NewInt(1),
```

Comment:
Related change in go1.19 - https://github.com/golang/go/commit/082cfabf126d63e952e1ac29d47c2a47f1c64bee